catalog/app/containers/Admin/Buckets/Buckets.tsx

```diff
@@ -1293,6 +1294,7 @@ function CRUD({ bucketName }: CRUDProps) {
   const history = RRDom.useHistory()
 
   const toolbarActions = [
+    <Table.Filter {...filtering} key="toolbar-filter" />,
```

Comment:
to me previous approach (passing extra controls as children) seemed somewhat cleaner, bc actions and arbitrary controls weren't mixed together